.github/workflows/cherry-pick-release-to-dev.yml

```diff
@@ -0,0 +1,29 @@
+# This job will automatically create a cherry pick PR from any release/* branch to the staging branch
+# It allows the staging branch to stay up-to-date with fixes made to specific release branches
+name: Cherry pick to staging
+on:
+  push:
+    branches:
+      - release/*
+jobs:
+  cherry_pick:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Create PR to branch
+        uses: gorillio/github-action-cherry-pick@master
+        with:
+          pr_branch: 'staging'
+        env:
+          GITHUB_TOKEN: ${{ secrets.OTTO_THE_BOT_GH_TOKEN }}
+
+      # Add the auto-merge flag
+      - name: Enable auto-merge
+        run: gh pr edit ${{ github.event.pull_request.id }} --add-label auto-merge
+        env:
+          PR_URL: ${{github.event.pull_request.html_url}}
+          GITHUB_TOKEN: ${{secrets.WEBTEAM_AUTOMERGE_TOKEN}}
```

Comment:
This part is currently broken. I'll fix it on the webapp side and probably re-enable it on other projects later
```suggestion
```